eslint.config.mjs

```diff
@@ -368,14 +351,12 @@ export default tsEslint.config(
 
   /* ******************** WEB PACKAGES ******************** */
   {
-    files: ['libs/design-system/**', 'libs/novui/**', 'apps/widget/**', 'apps/web/**'],
```

Comment:
removed web; now I think that we can even further improve this config as the web has it's own eslint config, so seems like some like airbnb can be removed